src/com/lftechnology/java/training/alina/jdbc/constants/Constants.java

```diff
@@ -0,0 +1,25 @@
+package com.lftechnology.java.training.alina.jdbc.constants;
+
+/**
+ * Common constants
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ *
+ */
+public class Constants {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)